bot.py

```diff
@@ -538,4 +540,41 @@ async def watchlist_history(interaction: nextcord.Interaction, watchlist_name):
     await interaction.response.send_message(response)
 
 
+
+"""
+Displays all the media that filtered based on tags in a watchlist.
+Args:
+    interaction: The interaction object representing the command invocation.
+    watchlist_name: The name of the watchlist to which the watched movies going to be displayed
+    tags: The tags that want to be based on to filter
+
+Returns:
```

Comment:
Good work adding in this documentation, I think you might have missed specifying the return type, good job though.

```suggestion
Returns:
None
```